content/learn/book/welcome/installation/_index.md

```diff
@@ -0,0 +1,118 @@
++++
+title = "Installing Rust and Bevy"
+weight = 2
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+I know you are itching to start making games, but we need to do a _small_ amount of setup first.
+
+## Rust Setup
+
+All Bevy app and engine code is written in Rust. This means that before we begin, we need to set up our Rust development environment.
+
+### Install Rust
+
+Install Rust by following the [Rust Getting Started Guide](https://www.rust-lang.org/learn/get-started).
+
+Once this is done, you should have the ```rustc``` compiler and the ```cargo``` build system installed in your path.
+
+### Install OS dependencies
+
+* [Linux](https://github.com/bevyengine/bevy/blob/main/docs/linux_dependencies.md)
+* Windows: Make sure to install [VS2019 build tools](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16)
```

Comment:
Now that we use `shaderc` instead of `bevy-glsl-to-spirv` you could also build for `x86_64-pc-windows-gnu` which doesn't require the `VS2019 build tools` unlike `x86_64-pc-windows-msvc`

Reply:
This should be in a separate issue <3. That section was already merged; I just targeted the wrong branch.

Reply:
Created #183